lib/commands/config/index.js

```diff
@@ -5,6 +5,16 @@ const Command           = require('../../command');
 const advancedOptions   = require('./advanced');
 
 class ConfigCommand extends Command {
+    static configureSubcommands(commandName, commandArgs, extensions) {
+        return commandArgs.command({
```

Comment:
Do you know if it's possible to expose these options when running `ghost config help`?

Reply:
Hmm...I didn’t test it but it should have. I’ll look into it.

Reply:
Looks like all I needed to do was add a description 😁